src/workerd/util/BUILD.bazel

```diff
@@ -1,7 +1,31 @@
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
 load("//:build/kj_test.bzl", "kj_test")
 load("//:build/wd_cc_library.bzl", "wd_cc_library")
 load("//:build/wd_cc_capnp_library.bzl", "wd_cc_capnp_library")
 
+bool_flag(
+    name = "workerd_perfetto",
+    build_setting_default = False,
+)
+
+config_setting(
+    name = "set_workerd_perfetto",
+    flag_values = {"workerd_perfetto": "True"},
+)
+
+wd_cc_library(
+    name = "perfetto",
+    srcs = ["tracing.c++"],
```

Comment:
My head would hurt less if the file names included the target name in them. ;) I used `perfetto-tracing.h/c++` internally, to try to keep the number of files named "trace.h" or "tracing.h", which is rather generic, low.